### PR TITLE
Perfdash - Extracting config paths to flags

### PR DESCRIPTION
**Pull request by krzysied**
- Adding support for multi flags
- Extracting hardcoded config paths to deployment params.

ref #546

### CODE REVIEW

File: perfdash/config.go [hunk 1]
```diff
@@ -239,12 +233,15 @@ type periodic struct {
...
-func getProwConfig() (Jobs, error) {
+func getProwConfig(configPaths []string) (Jobs, error) {
 	fmt.Fprintf(os.Stderr, "Fetching prow config from GitHub...\n")
 	jobs := Jobs{}
-	yamlLinks := []string{
-		"https://raw.githubusercontent.com/kubernetes/test-infra/master/config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml",
-		"https://raw.githubusercontent.com/kubernetes/test-infra/master/config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml",
+	yamlLinks := []string{}
+	for _, configPath := range configPaths {
+		//Currently perfdash supports only urls.
```
Comment (wojtek-t) [inline perfdash/config.go]: nit: whitespace after "//"

File: perfdash/config.go [hunk 2]
```diff
@@ -239,12 +233,15 @@ type periodic struct {
...
+		if strings.HasPrefix(configPath, "http") {
```
Comment (wojtek-t) [inline perfdash/config.go]: Maybe we should check extension...

File: perfdash/config.go (hunk 1 above)
Comment (krzysied) [reply perfdash/config.go]: Done.

File: perfdash/config.go (hunk 2 above)
Comment (krzysied) [reply perfdash/config.go]: Done.

**Pull request merged**